contracts/token/ZNSDomainToken.sol

```diff
@@ -174,6 +190,37 @@ contract ZNSDomainToken is
         return super.supportsInterface(interfaceId);
     }
 
+    /**
+     * @notice We override the standard transfer function to update the owner for both the `registry` and `token`
+     * This non-standard transfer is to behave similarly to the default transfer that only updates the `token`
+     * 
+     * @param from Owner of the token
+     * @param to Address to send the token to
+     * @param tokenId The token being transferred
+     */
+    function transferTokenFrom(address from, address to, uint256 tokenId) public override {
+        _transfer(from, to, tokenId);
+    }
+
+    /**
+     * @notice Override the standard transferFrom function to update the owner for both the `registry` and `token`
+     * 
+     * @dev See {IERC721-transferFrom}
+     */
+    function transferFrom(
+        address from,
+        address to,
+        uint256 tokenId
+    ) public override(ERC721Upgradeable, IERC721) {
```

Comment:
hm... what about all other transfer methods? there used to be 3 at least. those will not do what we want.

Reply:
Messaged you about this, but yes because the other two public `safeTransferFrom` both route through this function it's fine to just override this one.